docs/integrations/sources/hubspot.md

```diff
@@ -1,14 +1,43 @@
 # HubSpot
 
-This page guides you through setting up the HubSpot source connector.
+This page contains the setup guide and reference information for the [HubSpot](https://www.hubspot.com/) source connector.
 
-## Prerequisite
+## Prerequisites
 
-You can use OAuth or a Private App to authenticate your HubSpot account.
+You can either use OAuth or set up a Private App to authenticate your HubSpot account.
 
-In Airbyte Cloud, we highly recommend you use OAuth and not Private App authentication as it significantly simplifies the setup process.
+For Airbyte Cloud users, we highly recommend you use OAuth rather than Private App authentication, as it significantly simplifies the setup process. Conversely, for Airbyte Open Source users we recommend Private App authentication.
+
+More information on HubSpot authentication methods can be found 
+[here](https://developers.hubspot.com/docs/api/intro-to-auth).
+
+## Setup guide
+
+:::note
+Airbyte Cloud users authenticating via OAuth can skip straight to 
+[setting up the connector in Airbyte](#step-3-set-up-the-hubspot-source-connector-in-airbyte).  All other users must manually set up authentication and configure scopes.
+:::
+
+### Step 1: Set up the authentication method
+
+#### Private App setup (Recommended for Airbyte Open Source)
```

Comment:
Could you wrap oss or cloud-specific versions in `<!--env:[oss or cloud]-->` tags? See [example](https://github.com/airbytehq/airbyte/blob/master/docs/integrations/sources/airtable.md?plain=1#L21-L35) 

This makes it so that when we display these docs inside the app itself, we only display the sections between tags if they match the env e.g: `<!--env:oss-->` only shown on OSS and so on. 

this would make things like the `:::note` above redundant

Reply:
Love this feature! I've added the tags to the OAuth setup for OSS and the branching Authentication dropdown instructions during connector setup.

I left the Private App and Scope sections visible to both since Cloud users can still choose to use Private App auth. Would you prefer I made these oss specific as well? Not sure how strongly we want to veer them away from this path.